src/api/platform/models/api_drive_access.rs

```diff
@@ -0,0 +1,51 @@
+use std::fmt::{Display, Formatter};
+
+use serde::{Deserialize, Serialize};
+
+use super::ApiKeyId;
+
+#[derive(Debug, Deserialize, Serialize)]
+#[cfg_attr(feature = "strict", serde(deny_unknown_fields))]
+pub struct ApiDriveAccess {
+    user_key_id: ApiKeyId,
+    #[serde(rename = "bucket_id")]
+    drive_id: String,
```

Comment:
We have a dedicate type for this "ApiDriveId". While this is effectively a String, we're using it as a restrictive API type to ensure you're using the correct ID in the correct locations. This also provides us the ability to centrally modify this type if we want to change the representation to something more specific (A UUID type would be a good future change for that underlying type).